api/v1alpha1/usersignup_types.go

```diff
@@ -36,6 +36,10 @@ const (
 	// Activations are counted after phone verification succeeded
 	UserSignupActivationCounterAnnotationKey = LabelKeyPrefix + "activation-counter"
 
+	// UserSignupOriginalSubAnnotationKey is used to set a temporary annotation value in order to migrate users
+	// to an SSO provider with the user's original UserID value as the sub claim
+	UserSignupOriginalSubAnnotationKey = LabelKeyPrefix + "original-sub"
```

Comment:
You could use the existing annotation key:
```suggestion
	UserSignupOriginalSubAnnotationKey = UserAccountOriginalSubAnnotationKey
```

Reply:
PR has been rewritten to introduce new property values rather than use annotations